src/analysis/raster/qgsrastercalcnode.h

```diff
@@ -125,6 +126,13 @@ class ANALYSIS_EXPORT QgsRasterCalcNode
 
     static QgsRasterCalcNode *parseRasterCalcString( const QString &str, QString &parserErrorMsg ) SIP_FACTORY;
 
+    /**
+     * Returns a dictionary that has the raster name as keys and the raster references as values
+     * (it's possible to have different value as one raster layer has different bands)
+     * \note since QGIS 3.22
+     */
+    static QStringList referencedLayerNames( const QString &str ) SIP_FACTORY;
```

Comment:
please remove the SIP_FACTORY annotation - this is used when we return a pointer to an object and its ownership is passed to the caller.